pkg/resources/satellite/pod/node-pod.yaml

```diff
@@ -29,6 +37,8 @@ spec:
         - mountPath: /usr/src
           name: usr-src
           readOnly: true
+        - mountPath: /tmp
+          name: tmp
```

Comment:
This means that the module loader and satellite now use the *same* `/tmp`, right? I guess that is fine.

Reply:
Yes. I didn't really think about it, but a separate /tmp may make sense.

On the other hand, from previous tests I know that kubernetes doesn't clean up these emptyDir volumes once they become unused, but only after the whole pod is removed. So while it does provide more separation, it does not disk usage.

Reply:
> a separate /tmp may make sense.

Perhaps more logical, but irrelevant in practice, I would say. The module loader does use static paths `/tmp/pkg` and `/tmp/ko`, but the satellite certainly won't conflict with those or be affected by their presence, so we're fine.